code/__DEFINES/__game.dm

```diff
@@ -95,6 +96,12 @@
 /// Is something in the IC chat filter? This is config dependent.
 #define CHAT_FILTER_CHECK(text) (config.ic_filter_regex && findtext(text, config.ic_filter_regex))
 
+//for whether AI eyes see static, and whether it is mouse-opaque or not
```

Comment:
I still don't know why this is counting as an alteration, or if I need to fix it.